vms/platformvm/state/disk_staker_diff_iterator.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+const (
+	// diffKey = [subnetID] + [inverseHeight] + [nodeID]
+	startDiffKeyLength  = ids.IDLen + database.Uint64Size
+	diffKeyLength       = startDiffKeyLength + ids.NodeIDLen
```

Comment:
nit: The `diffKey` comment was a bit confusing since it's referencing 3 elements but L16 only has 2 elements. I would suggest moving the comment down one line to be above the `diffKeyLength` if that is what it's referencing. Just personal preference though.

Reply:
Now we have a separate comment for `startDiffKey` and `diffKey`